proposals/0293-vehicle-type-filter.md

```diff
@@ -436,9 +475,9 @@ Above mentioned changes need to be implemented in SDL Core, the Java Suite proxy
 ## Alternatives considered
 
 ### Using RPC instead of Protocol message to share vehicle type info
-1. As per the current implementation, the RAI is the first RPC sent to an SDL enabled IVI system and the vehicle type information is available to the app in its response. 
-2. If `GetVehicleType` is an RPC to receive the vehicle type info, it would be the first RPC to be used. Since, the app should be able to use the `GetVehicleType` RPC to know vehicle type info before the app registration on IVI.
-3. Since the RPC message version is sent to the app in `RAI response`, when connected to an older SDL enabled IVI, the app  would not know if the `GetVehicleType` RPC is supported or not.
+1. As per the current implementation, the RAI is the first RPC sent to an SDL enabled IVI system, and the vehicle type information is available to the app in its response. 
+2. To know vehicle type details before the Register App interface, we have to introduce new RPC `GetVehicleType` to receive the vehicle type info. The App will have to send `GetVehicleType` before RAI. Since the app needs to know vehicle type info before the app registration on IVI.
```

Comment:
```suggestion
2. To know vehicle type details before the Register App Interface, we have to introduce new RPC `GetVehicleType` to receive the vehicle type info. The App will have to send `GetVehicleType` before RAI. Since the app needs to know vehicle type info before the app registration on IVI.
```